src/Entity/User.php

```diff
@@ -62,8 +64,11 @@
  * @UniqueEntity(fields={"usernameCanonical"}, errorPath="username")
  * @UniqueEntity("facebookId")
  */
-class User extends BaseUser implements JWTUserInterface, ChannelAwareInterface
+class User extends BaseUser implements TaggableInterface
```

Comment:
Actually, we want to tag customers, not users. A person ordering always maps to a customer, but not necessarily to a user.